lib/driver.js

```diff
@@ -578,6 +585,11 @@ class XCUITestDriver extends BaseDriver {
       await this.stopHttpsAsyncServer();
     }
 
+    if (this.mjpegStream) {
+      log.info('Closing MJPEG stream');
+      this.mjpegStream.stop();
```

Comment:
await ?

Reply:
Doesn't appear to be an async functions (https://github.com/appium/appium-support/blob/master/lib/mjpeg.js#L145)